.travis.yml

```diff
@@ -0,0 +1,16 @@
+dist: trusty
+sudo: false
+language: python
+python:
+    - "3.4"
```

Comment:
shouldn't we test against 3.6? We use 3.6 in our dockerfile, no?

Reply:
Yes, this was a hang over from when we were using the standard GCE image. Will change.